Source/gmenu.cpp

```diff
@@ -144,7 +144,7 @@ static void gmenu_left_right(bool isRight)
 		return;
 
 	step = sgpCurrItem->dwFlags & 0xFFF;
-	steps = (int)(sgpCurrItem->dwFlags & 0xFFF000) >> 12;
+	steps = (int)(sgpCurrItem->dwFlags & 0xFFF000) / 4096;

```

Comment:
Semantically this is not a division but is an extraction of a value from a non-arithmetic type (same below)